test/hotspot/gtest/concurrentTestRunner.inline.hpp

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
for c++, we don't use camelCase in filenames, but rather use small_snake_case

Reply:
OK. I saw gtestMain.cpp, gtestLauncher.cpp and a few others, and just followed that. I also see a number of test_camelCase.cpp: test_primitiveConversions.cpp, test_logSelectionList.cpp and so on. In fact, it seems the most prevalent pattern for gtests is test_camelCase.cpp.

Anyway, no problem, I can rename this file to concurrent_test_runner.inline.hpp

Reply:
This statement that we use small_snake_case for filenames is mostly wrong.  Most HotSpot files contain some primary class, whose name is CamelCased, and the associated files are camelCased.  There are a fairly small number of exceptions to that.  There are some suffixes and prefixes that get added on, such as cpu and os suffixes (`_linux`, `_windows`, &etc, suffixes, `test_` gtest prefix, component prefixes like `c1_` and `gc_`), but those are mostly add-ons to the basic camelCase convention.

Reply:
apparently, I had a false memory, we indeed have more `camelCase` than `small_snake_case` files, so @mseledts, you will need to revert that. sorry for wasting your time. in my defense, most of the files are just `case` hence can be interpreted either way :)

Reply:
No worries. I will 'git rename' it back to the original name, I hope this will work.

Reply:
See also "Naming and Grouping" here: https://github.com/openjdk/jdk/blob/master/doc/hotspot-unit-tests.md